pkg/cert/util.go

```diff
@@ -0,0 +1,44 @@
+package cert
```

Comment:
Does it deserve it's own package? or should this be in pkg/cert.go since util files are usually an anti-pattern

Reply:
Some functions in this package indeed belong in crypto. This util file should go. I'll move functions from this existing package to crypto in a seperate PR and then this file will be gone anyways.